e2e/integration/utils/minio.js

```diff
@@ -0,0 +1,35 @@
+const nconf = require('nconf');
+const minio = require('minio');
+const { getEnv } = require('./clients');
+const fs = require('fs');
+
+module.exports.getObjectContentFromMinio = async objectName => {
+  nconf.required(['MINIO_HOST', 'MINIO_PORT', 'MINIO_BUCKET']);
+
+  const mc = new minio.Client({
+    endPoint: nconf.get('MINIO_HOST'),
+    port: Number(nconf.get('MINIO_PORT')),
+    accessKey: getEnv('MINIO_ACCESS_KEY'),
+    secretKey: getEnv('MINIO_SECRET_KEY'),
+    secure: false,
+  });
+
+  return new Promise((resolve, reject) =>
+    mc.getObject(nconf.get('MINIO_BUCKET'), objectName, (err, stream) => {
```

Comment:
check promisify or bluebird stream utils

Reply:
I checked it.. It doesn't exist